docs/content/_navigation.json

```diff
@@ -241,6 +241,10 @@
           {
             "title": "Asset Materializations",
             "path": "/concepts/assets/asset-materializations"
+          },
+          {
+            "title": "Software-Defined Assets",
+            "path": "/concepts/assets/software-defined-assets"
```

Comment:
I think that the software-defined assets page should be placed above asset materializations page in the navbar

Reply:
or we can have an "assets" section that keeps everything in terms of ops (containing asset materializations and asset observations), and a "software defined assets" section that contains this page

Reply:
cool, I took your first suggestion and moved it up